### PR TITLE
optimize some code

### PR DESCRIPTION
**Pull request by slievrly**
Motivation:

Explain here the context, and why you're making that change.
What is the problem you're trying to solve.

Modification:

Optimize  condition

Optimize collections

Remove unnecessary semicolon

Class initializer can be static

### CODE REVIEW

File: transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
  * it only works on linux.
  */
 public final class EpollEventLoopGroup extends MultithreadEventLoopGroup {
-    {
+    static {
```
Comment (normanmaurer) [inline transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java]: I think there was reason for not using static... @carl-mastrangelo do you remember ?

Comment (carl-mastrangelo) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java]: Delaying initialization for out-of-band static init.  If System.loadLibrary is called after EpollEventLoopGroup is referenced, there's no way for the Native methods to be linked correctly.

Comment (slievrly) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java]: Thank you, I will learn this point.

Comment (normanmaurer) [reply transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoopGroup.java]: Please revert

File: transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventLoopGroup.java [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
 @UnstableApi
 public final class KQueueEventLoopGroup extends MultithreadEventLoopGroup {
-    {
+    static {
         // Ensure JNI is initialized by the time this class is loaded by this time!
```
Comment (normanmaurer) [inline transport-native-kqueue/src/main/java/io/netty/channel/kqueue/KQueueEventLoopGroup.java]: Please revert

**Comment by slievrly** [comment]
@normanmaurer revert done.

**Comment by normanmaurer** [comment]
@netty-bot test this please

**Comment by slievrly** [comment]
https://ci.netty.io/job/netty-centos6-java8-prb/1220/testReport/junit/io.netty.channel.pool/FixedChannelPoolTest/testAcquireNewConnection/

**Pull request merged**